scripts/asdf2devbox.py

```diff
@@ -27,6 +24,7 @@ def get_installed_version(package):
                 "bosh": "bosh-cli",
                 "cf": "cloudfoundry-cli",
                 "concourse": "fly",
+                "credhub": "credhub-cli",
```

Comment:
Why do we need this mapping even though there is no `credhub` defined in https://github.com/cloudfoundry/app-autoscaler-release/blob/main/.tool-versions ?

Reply:
removed in ea75773c94a8fc8eee28bf253c2c58ee45e1b186